src/main/resources/db/migration/ledgersyncagg/V1_4_1__add-indexes.sql

```diff
@@ -1,21 +1,9 @@
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_block_time2_idx
-    ON address_tx_amount (unit, to_timestamp(block_time));
 CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_idx
     ON address_tx_amount (stake_address);
 CREATE INDEX IF NOT EXISTS address_tx_amount_epoch_idx
     ON address_tx_amount (epoch);
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_quantity_block_time_idx
-    ON address_tx_amount (unit, quantity, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_tx_hash_block_time_idx
-    ON address_tx_amount (unit, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_address_tx_hash_block_time_idx
-    ON address_tx_amount (address, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_tx_hash_block_time_idx
-    ON address_tx_amount (stake_address, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_unit_block_time_idx
-    ON address_tx_amount (stake_address, unit, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_slot_idx
-    ON address_tx_amount (slot);
+CREATE INDEX IF NOT EXISTS address_tx_amount_block_time_idx
+    ON address_tx_amount (block_time);
```

Comment:
do we need this?

Reply:
@Kammerlo @Sotatek-DucPhung do you guys we could create an index here on

(unit, slot, quantity) to get top 20 ada/token holders for address (And do the same for stake_address_balance) ?

Reply:
We could do some tests to see if it increases the speed to something acceptable. Currently when calculating the top ada holders on these tables it will take minutes on mainnet. I'd give it a try.